translate/lang/ro.py

```diff
@@ -47,17 +47,22 @@ def contains_illegal(self, illegal_sequence, target_str):
 
     @cosmetic
     def cedillas(self, str1, str2):
-        """
-        Checks if the translation strings contains an illegal cedilla character
-          illegal_chars = [u'Ţ', u'Ş', u'ţ', u'ş']
-        Cedillas are obsoleted diacritics for Romanian and should never be used
-          where the target string is Romanian.
-        Cedilla-letters are only valid for Turkish (S-cedilla)
-          and Gagauz languages (S-cedilla and T-comma)
-        Fun fact: Gagauz is the only known language to use T-cedilla
-               :param str1: the source string
-               :param str2: the target (translated) string
-               :return: True if str2 contains a cedilla character
+        """Check if the translation contains an illegal cedilla character
+
+        Cedillas are obsoleted diacritics for Romanian:
+
+          - U+0162 Latin capital letter T with cedilla
+          - U+0163 Latin small letter T with cedilla
+          - U+015E Latin capital letter S with cedilla
+          - U+015F Latin small letter S with cedilla
```

Comment:
Haha, that's a hack of note :) Now why didn't I think of that.  I'm good with this.